diff -Nuard gtksourceview-sharp-0.5.orig/configure.in gtksourceview-sharp-0.5/configure.in
--- gtksourceview-sharp-0.5.orig/configure.in	2004-06-29 21:13:06.000000000 +0200
+++ gtksourceview-sharp-0.5/configure.in	2004-07-04 09:12:09.971542680 +0200
@@ -57,7 +57,7 @@
 	AC_MSG_ERROR([No gacutil tool found])
 fi
 
-GACUTIL_FLAGS='/package gtk-sharp /gacdir $(libdir)'
+GACUTIL_FLAGS='/package gtk-sharp /root $(DESTDIR)$(prefix)/lib'
                                      
 AC_SUBST(RUNTIME)
 AC_SUBST(CSC)
diff -Nuard gtksourceview-sharp-0.5.orig/makefile.am gtksourceview-sharp-0.5/makefile.am
--- gtksourceview-sharp-0.5.orig/makefile.am	2004-06-26 23:39:15.000000000 +0200
+++ gtksourceview-sharp-0.5/makefile.am	2004-07-04 09:12:09.971542680 +0200
@@ -5,7 +5,7 @@
 extra_langdir= @gtksourceview_prefix@/share/gtksourceview-1.0/language-specs
 extra_lang_DATA=	nemerle.lang vbnet.lang
 
-pkgconfigdir = @prefix@/lib/pkgconfig
+pkgconfigdir = @libdir@/pkgconfig
 pkgconfig_DATA = gtksourceview-sharp.pc
 
 # should be the same as the gnome prefix
