diff -aurN gtksourceview-sharp-0.7.orig/configure.in gtksourceview-sharp-0.7/configure.in
--- gtksourceview-sharp-0.7.orig/configure.in	2005-03-12 10:54:22.000000000 +0100
+++ gtksourceview-sharp-0.7/configure.in	2005-04-21 20:38:55.000000000 +0200
@@ -55,7 +55,7 @@
 	AC_MSG_ERROR([No gacutil tool found])
 fi
 
-GACUTIL_FLAGS='/package gtksourceview-sharp /gacdir $(libdir)'
+GACUTIL_FLAGS='/package gtksourceview-sharp /root $(DESTDIR)$(libdir)'
                                      
 AC_SUBST(RUNTIME)
 AC_SUBST(CSC)
